go/oasis-node/cmd/stake/account.go

```diff
@@ -22,6 +23,9 @@ const (
 	// CfgAccountAddr configures the account address.
 	CfgAccountAddr = "stake.account.address"
 
+	// CfgAccountAddr configures the entity ID.
+	CfgEntityID = "entity_id"
```

Comment:
I would rather name this "public key" or something as it can be either an entity or a node public key (nodes also have balances, e.g., to pay fees).

Reply:
Agreed.